src/Symfony/Component/Security/Core/Util/ClassUtils.php

```diff
@@ -27,6 +27,8 @@ class ClassUtils
     /**
      * Marker for Proxy class names.
      *
+     * @deprecated use doctrine classutils instead
```

Comment:
and convention is: `@deprecated since version 2.8, to be removed in 3.0. Use Acl ClassUtils instead.`